static/js/controllers/messages-content.js

```diff
@@ -235,5 +242,8 @@ angular.module('inboxControllers').controller('MessagesContentCtrl',
       }
     });
 
+    RecurringProcessesManager.startProcessByName(
+      UPDATE_RELATIVE_TIME
+    );
```

Comment:
I think it might be better to just register/destroy this in InboxCtrl because most pages will have a relative date somewhere on them and that means we won't miss a registration or destroy step on one page.